### ISSUE TITLE
setItemsList from npm README.md

### ISSUE THREAD

**Issue by gabrielnaoto** [LauraBeatris/use-selected-items-hook#18]
First of all, I think this lib is so useful! 🚀 

Secondly, I think that npm docs differ from the github ones. Initially I was not sure which one was more up to date.

And last but not least, I saw in the npm docs that we could've had a `setItemsList` action that would allow us to change the items list, but after checking github and the source code I noticed that's missing.

I was thinking why don't we have a way to change the items list after initialization? Let's say you want to retrieve your items from a backend request, or after a user perfom some action in the screen. Shouldn't we be able to change tha programmatically? Am I missing something here?

**Comment by gabrielnaoto** [comment]
There's an effect for the `initialItems`, right? It just didn't work because I was passing an invalid argument because I was using the wrong docs.

**Comment by LauraBeatris** [comment]
Thanks for creating the issue!

The `README.md` is just updated when a new version is published https://docs.npmjs.com/about-package-readme-files#updating-an-existing-package-readme-file

But it was being cached and it didn't update when I published the last major version (2.1)

I updated the `README.md` by cleaning the cache and publishing a patch
```bash
npm cache clean --force
npm version patch
npm publish
```

Now if you go to the NPM docs https://www.npmjs.com/package/use-selected-items-hook, it's up to date with the GitHub one. Sorry for the inconvenient.
